GH actions throws error `undefined: accumAVX512`

I am trying to use this library v1.0.0-rc3 and when I try to build the binary on GH Actions, I am getting the following error. Is there a way to avoid this error?

```
Error: vendor/github.com/zeebo/xxh3/hash128.go:244:3: undefined: accumAVX512
Error: vendor/github.com/zeebo/xxh3/hash64.go:174:3: undefined: accumAVX512
```

https://github.com/kubeops/csi-driver-cacerts/runs/4051947892?check_suite_focus=true

--- a/accum_stubs_other.go
+++ b/accum_stubs_other.go
@@ -13,10 +13,11 @@ const (
 	hasAVX512 = false
 )
 
-func accumAVX2(acc *[8]u64, data, key unsafe.Pointer, len u64) { panic("unreachable") }
-func accumSSE(acc *[8]u64, data, key unsafe.Pointer, len u64)  { panic("unreachable") }
-func accumBlockAVX2(acc *[8]u64, data, key unsafe.Pointer)     { panic("unreachable") }
-func accumBlockSSE(acc *[8]u64, data, key unsafe.Pointer)      { panic("unreachable") }
+func accumAVX2(acc *[8]u64, data, key unsafe.Pointer, len u64)   { panic("unreachable") }
+func accumSSE(acc *[8]u64, data, key unsafe.Pointer, len u64)    { panic("unreachable") }
+func accumBlockAVX2(acc *[8]u64, data, key unsafe.Pointer)       { panic("unreachable") }
+func accumBlockSSE(acc *[8]u64, data, key unsafe.Pointer)        { panic("unreachable") }
+func accumAVX512(acc *[8]u64, data, key unsafe.Pointer, len u64) { panic("unreachable") }
 
 func withAVX2(cb func())    { cb() }
 func withSSE2(cb func())    { cb() }

